supertokens_python/recipe/thirdparty/providers/utils.py

```diff
@@ -1,4 +1,4 @@
-from typing import Any, Dict, Optional
+from typing import Any, Dict, Optional, Tuple
```

Comment:
what version is Tuple supported in?

Reply:
`3.5`

https://docs.python.org/3/library/typing.html#typing.Tuple